firmware/brake/kia_soul_petrol/src/init.cpp

```diff
@@ -61,4 +62,10 @@ void init_communication_interfaces( void )
 
     DEBUG_PRINT( "init Control CAN - " );
     init_can( g_control_can );
+
+    // Filter CAN messages - accept if (CAN_ID & mask) == (filter & mask)
+    g_control_can.init_Mask(0, 0, 0x7F0); // Filter for 0x0N0 to 0x0NF
```

Comment:
:100: for omitting the leading and trailing spaces inside parens :)

Reply:
Oops. As much as I'd like to not have the spaces, the style guide says we must.

7d665ba